### ISSUE TITLE
'null' is not an object

### ISSUE THREAD

**Issue by kingsleyh** [danyg/jasmine-durandal#2]
Hi

I'm trying to use this plugin with the latest version of Durandal but getting this error when I try to run a spec using it. 

I've downloaded the html starter kit pro from https://github.com/RainerAtSpirit/HTMLStarterKitPro

I've just copied the jasmine-durandal-1.3x.js into lib/durandal/js/plugins and then copied the howtouse spec (just the welcome,flickr and shell) 

Testing jasmine specs via phantom
..............>> TypeError: 'null' is not an object (evaluating 'allElements.length') at
>> lib/durandal/js/viewEngine.js:93 
>> lib/durandal/js/viewEngine.js:84 
>> lib/durandal/js/viewEngine.js:131 
>> lib/jquery/jquery-1.9.1.js:1192 
>> lib/jquery/jquery-1.9.1.js:1037 
>> lib/jquery/jquery-1.9.1.js:1148 
>> lib/jquery/jquery-1.9.1.js:1237 
>> lib/durandal/js/system.js:256

**Comment by kingsleyh** [comment]
I havent tried to upgrade to the latest durandal version - I will just give that a go also

**Comment by danyg** [comment]
Sorry for the moment only durandal 2.0.1 is supported

I have planed to make the needed changes to support the latest durandal
version. But for the moment I can't take it.

**Comment by kingsleyh** [comment]
Hi - yeah upgrading didnt work either. Howerver I also grabbed the 2.0.1 code and put it in lib/durandal but this gives the same error as above.

**Comment by kingsleyh** [comment]
it looks like the durandal module is not being located by the acquire function in system.js on line 256.

**Comment by kingsleyh** [comment]
I'm totally confused about how to get this working? I cloned your repo and ran npm install and then grunt tdd and all the tests passed. 

I copied the durandal folder from your project to mine - but I still get the same error.

Am I missing something?

**Comment by kingsleyh** [comment]
ok - finally go this working. In my Gruntfile I was using: grunt-template-jasmine-requirejs
but I noticed your one has grunt-template-jasmine-requirejs-preloader instead. Once I switched to using the preloader one - everything started working. I don't know exactly why this is the case - I can only assume that some stuff needs to be pre-loaded before the specs can run properly. 

A second thing I noticed is that I can't have jasmine-durandal tests and regular durandal jasmine tests in the same directory or I get all sorts of model not required issues. But if I keep them all separated and with different grunt tasks to run them it's all fine.

--K
